### PR TITLE
docs(GSG): refresh docs for 2.5 release

### PR DESCRIPTION
**Pull request by bwendlandt-intel**
## PR Checklist

- [x] Leverages formatting as provided by https://squidfunk.github.io/mkdocs-material/reference/abbreviations/ 
- [x] Have viewed and verified edits in rendered form ensuring proper formatting

## What are you changing?

- Update GSG for 2.5
- Small wording for Wireless
- Postgres K8S change
- Swap Video

### CODE REVIEW

File: docs/GetStarted/buildRPC.md [hunk 1]
```diff
@@ -132,8 +132,8 @@ The toolkit provides a reference implementation called the Sample Web UI to mana
         </figure>
 
 
-    !!! error "Troubleshooting"
-        Run into an issue? Try these [troubleshooting steps](../Reference/troubleshooting.md).
+        !!! error "Troubleshooting"
```
Comment (rsdmike) [inline docs/GetStarted/buildRPC.md:135]: I don't think you meant to put this inside of success?

image

File: docs/GetStarted/setup.md [hunk 2]
```diff
@@ -123,25 +123,23 @@ Set the shared secret used in Kong for JWT authentication.
     !!! success
         ``` bash
         IMAGE                               STATUS                        NAMES
-        intel/oact-rps:v2.2.0               Up 2 minutes (healthy)        open-amt-cloud-toolkit_rps_1      
+        intel/oact-rps:latest               Up 2 minutes (healthy)        open-amt-cloud-toolkit_rps_1
```
Comment (rsdmike) [inline docs/GetStarted/setup.md:126]: are you able to use the mkdocs versions here to plug it in?

File: docs/GetStarted/setup.md [hunk 3]
```diff
@@ -123,25 +123,23 @@ Set the shared secret used in Kong for JWT authentication.
...
+        intel/oact-webui:latest             Up 2 minutes                  open-amt-cloud-toolkit_webui_1    
         kong:2.3                            Up 2 minutes (healthy)        open-amt-cloud-toolkit_kong_1     
-        intel/oact-mps:v2.2.0               Up 2 minutes (healthy)        open-amt-cloud-toolkit_mps_1
+        intel/oact-mps:latest               Up 2 minutes (healthy)        open-amt-cloud-toolkit_mps_1
         ```
-
   
-!!! important 
-    ** Container Issues **
+        !!! warning "Warning - Container Issues"
```
Comment (rsdmike) [inline docs/GetStarted/setup.md:136]: Warning shouldn't be inside success

image

**Review by rsdmike — CHANGES_REQUESTED** [review]

File: docs/GetStarted/setup.md (hunk 2 above)
Comment (bwendlandt-intel) [reply docs/GetStarted/setup.md:126]: closing since irrelevant as latest is correct

**Review by rsdmike — APPROVED** [review]

**Pull request merged**